### DISCUSSION TITLE
Page immediately reloads after pressing "Process my application".

### DISCUSSION THREAD

**Discussion by jaac-dev** [community/community#111038] [GitHub Education]
### Hi everyone,

Hi,

Every time I try and submit the image of my transcript the page immediately reloads and provides no error. I've check the size and resolution of the image, and they both fit the requirements listed on the page.

**Comment by SebastienXia** [comment]
Hi @jackkdev 

I had the same problem, the file download does not work, you just have to take a photo with your phone to be able to finish the process

close the discussion if this helped you with your problem

**Reply by SebastienXia** [reply]
You have 2 options one with the file download (which does not work) and the other option which is to take a photo of the proof to be able to finish the process.

**Reply by jaac-dev** [reply]
Hey, thanks for the reply and solution!
 
(Also, yes I'm aware, I've gone through the verification process a few times over the last 6ish years).

I'll go ahead and close this soon, but the file upload system not working is very disappointing and honestly kind of ridiculous. It didn't work for me back in January either, and I just left it at that since I didn't really need any of the education pack features (ie. GitKraken).
